### PR TITLE
Added Project Democracy app

### PR DESCRIPTION
**Pull request by SkiingIsFun123**
Added Project Democracy app

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -4872,6 +4872,21 @@
...
+      "category-ids": [
+        "news"
+      ],
+      "source": "https://github.com/SkiingIsFun123/Project-Democracy",
+      "tags": [
+        "swift"
+      ],
+      "itunes": "https://apps.apple.com/us/app/project-democracy/id6443437676?uo=4",
+      "license": "other",
+      "date_added": "October 7, 2022,
```
Comment (dkhamsing) [inline contents.json:4888]: can you add a `"` after 2022

File: contents.json [hunk 2]
```diff
@@ -4872,6 +4872,21 @@
...
+      "suggested_by": "@SkiingIsFun123",
```
Comment (dkhamsing) [inline contents.json:4889]: Can you remove the `,` at the end

**Comment by SkiingIsFun123** [comment]
Made the changes

**Comment by dkhamsing** [comment]
Thx!

**Pull request merged**